SECURITY.md

```diff
@@ -0,0 +1,30 @@
+# Security Policy
+
+## Supported Versions
+
+Kured follows the following version scheme:
```

Comment:
"follows the following" has a bit of a stutter. We can probably link to the semver spec. Maybe something like:

Kured releases versions in accordance with the [Semantic Version spefication](https://semver.org):